ragna/deploy/_ui/styles.py

```diff
@@ -11,109 +11,69 @@ def divider():
 
 
 def apply_design_modifiers():
+    apply_design_modifiers_global()
     apply_design_modifiers_source_accordion()
-    # add here calls to other design modifiers,
-    #   group them per UI component
+    apply_design_modifiers_auth_page()
+    apply_design_modifiers_central_view()
+    apply_design_modifiers_chat_info()
+    apply_design_modifiers_chat_interface()
+
+
+def add_modifier(
+    modifier_class: Type[Any], modifications: Any, property: str = "stylesheets"
```

Comment:
Are we ever passing something for `property`? If not can we just hardcode it and maybe rename the function `add_stylesheets_modifier`?

Reply:
We're not yet, but changes are high that we will. Modifiers are very powerful, we can override any property of a Panel widget/viewable object. `stylesheets`, but also `css_classes`, or `margin`, or whatever. I'm unsure which we'll use in the future, as I really heavily on `stylesheets` as it's the most versatile, but I'd like to keep this option.